plugins/auth-backend/src/service/router.ts

```diff
@@ -249,3 +249,21 @@ export function createOriginFilter(
     return allowedOriginPatterns.some(pattern => pattern.match(origin));
   };
 }
+
+export function getDefaultBackstageTokenExpiryTime(config: Config) {
+  const processingIntervalKey = 'auth.backstageTokenExpiration';
+
+  if (!config.has(processingIntervalKey)) {
+    return BACKSTAGE_SESSION_EXPIRATION;
+  }
+
+  const duration = readDurationFromConfig(config, {
+    key: processingIntervalKey,
+  });
+  const seconds = Math.max(
+    1,
```

Comment:
One second is unreasonably low, given that refresh loops will be trying to keep this one updated - they will be spinning infinitely and possibly hurting both frontend and backend performance.

https://github.com/backstage/backstage/blob/b6fcdffd7f21f8df644cf7365e9ab7113883fdb5/packages/core-app-api/src/apis/implementations/auth/oauth2/OAuth2.ts#L156

The current margin is five minutes (five minutes before actual expiry, a refresh can start). The value here should probably be something more along the lines of ten minutes (600).